lib/src/main/java/com/auth0/msg/AlgorithmEnum.java

```diff
@@ -0,0 +1,11 @@
+package com.auth0.msg;
+
```

Comment:
Add javadoc. Also Justin and you should share the enum Algorithm enum.

@jdahmubed - please use just one enum for algorithms. At this time, you an Leo have two different copies.
@lccodes FYI

Reply:
@jdahmubed if you can import and use my AlgorithmEnum we should just go with that

Reply:
i dont agree w including "Enum" in the enum name.  mine leaves it out.